bin/c-mk-start.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+const { exec } = require('shelljs');
+
+// The basic program, which uses sub-commands.
+program
+    .parse(process.argv);
+
+exec('minikube start --extra-config=apiserver.ServiceNodePortRange=80-32767 --memory=4096 --vm-driver=vmwarefusion');
```

Comment:
@smebberson Just curious, would there be any benefit in changing this to use the hyperkit driver?
https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver